demo/app/background-service.ts

```diff
@@ -0,0 +1,41 @@
+import * as geolocation from "nativescript-geolocation";
+import { Accuracy } from "ui/enums";
+let Toast = require("nativescript-toast");
+(<any>android.app.Service).extend("com.nativescript.location.BackgroundService", {
+    onStartCommand: function (intent, flags, startId) {
```

Comment:
The service shows toasts only on initial app deploy. If the app has been stopped, no more toast will be displayed. Additionally, it would be nice to be able to stop the toasts once started.